modules/ipi-install-replacing-a-bare-metal-control-plane-node.adoc

```diff
@@ -17,16 +17,17 @@ Existing control plane `BareMetalHost` objects may have the `externallyProvision
 
 .Procedure
 
-. Remove the old `baremetalhost`, `machine` and `node` objects:
+. Remove the old `BareMetalHost` and `Machine` objects:
 +
 [source,terminal]
 ----
-$ oc delete node kni1-vmaster-0
 $ oc delete bmh -n openshift-machine-api vmaster-0
 $ oc delete machine -n openshift-machine-api kni1-master-0
 ----
 +
-. Create the new `baremetalhost` object and the secret to store the BMC credentials:
+After you remove the `BareMetalHost` and `Machine` objects, then the Bare Metal Operator automatically deletes the `Node` object.
```

Comment:
FWIW it is the Machine controller and not the baremetal-operator that deletes the Node.